contracts/eth/nearbridge/contracts/NearBridge.sol

```diff
@@ -41,8 +41,8 @@ contract NearBridge is INearBridge, AdminControlled {
     bytes32 untrustedHash;
```

Comment:
The code looks good :)
As we are not longue use a watchdog we can rename those variables that has "untrusted" prefix by removing it.
Also check if those variables are used, especially if they was used by the watchdog functions.